tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -32,10 +32,13 @@ import (
 var testBucket = flag.String("testbucket", "", "The GCS bucket used for the test.")
 
 var (
-	testDir string
-	binFile string
-	logFile string
-	mntDir  string
+	testDir  string
+	binFile  string
+	logFile  string
+	mntDir   string
+	tmpDir   string
+	fileName string
+	err      error
```

Comment:
fileName and err shouldn't be global variables. Make the fileName as return parameter for the function call. 
For the err, i can see it not be need to sent back to the caller. So you can make it a local variable inside the method.

Reply:
Done